src/pages/Rules.jsx

```diff
@@ -28,12 +32,10 @@ const Rules = () => {
                 <article className={`card-dim card-shadow-xs ${theme === 'dark' && 'b-solid b-secondary'} pd-md`}>
                     <ul>
                         {
-                            rulesState?.rules?.map((rule, index) =>
-
+                            rulesState?.map((rule, index) =>
```

Comment:
You can store the document id obtained from firestore in each rule when you fetch them and then you can use that id here instead of using index. I hope I make sense here

Reply:
Ah! yes. I missed out on this. Thanks for reminding.